db/models/feedback.js

```diff
@@ -0,0 +1,48 @@
+// @ts-nocheck
```

Comment:
This is the main change to 👀 at. Updates the Feedback and Survey models to use classes which is now the default in v5 (which we should upgrade to soon, but this works already in v4).